### PR TITLE
bump dependencies

### PR DESCRIPTION
**Pull request by PeterDaveHello**
bump dependencies without modifying the codes and still keep the origin functions
- fs-extra ^0.16.5  ->  ^0.24.0
- fs-plus    2.7.0  ->    2.8.1
- glob      ^5.0.3  ->  ^5.0.14
- gry       ^2.1.0  ->   ^4.0.0
- minimist  ^1.1.1  ->   ^1.2.0
- oargv     ^1.2.0  ->   ^2.0.0
- mkdirp    ^0.5.0  ->   ^0.5.1
- tmp      ^0.0.25  ->  ^0.0.27
- ul        ^2.1.0  ->   ^5.0.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,16 +32,16 @@
...
-    "minimist": "^1.1.1",
-    "oargv": "^1.2.0",
-    "mkdirp": "^0.5.0",
-    "tmp": "^0.0.25",
-    "ul": "^2.1.0"
+    "minimist": "^1.2.0",
+    "oargv": "^2.0.0",
+    "mkdirp": "^0.5.1",
+    "tmp": "^0.0.27",
+    "ul": "^5.0.0"
```
Comment (IonicaBizau) [inline package.json:44]: Hmm, this is going to break the functionality.

**Comment by IonicaBizau** [comment]
This is already (or should) be done in the new version. Please modify that. Thanks. :smile:

File: package.json (hunk 1 above)
Comment (PeterDaveHello) [inline package.json:44]: Which one?

Comment (IonicaBizau) [inline package.json:44]: The `ul`'s `merge` function (used here) was renamed into `deepMerge`. But please modify this in the `new-version` branch. Would be happy if you get it working and fix the issues @Piicksarn reported in #15. :dizzy:

**Pull request closed without merging**